src/Stripe.net/Services/Accounts/AccountService.cs

```diff
@@ -26,6 +26,16 @@ public AccountService(IStripeClient client)
 
         public override string BasePath => "/v1/accounts";
 
+        public virtual ListAccountCapability Capabilities(string id, AccountCapabilitiesOptions options = null, RequestOptions requestOptions = null)
```

Comment:
Flagging this should not be here, we don't do nested on the parent service in dotnet for most cases except a handful. We might in the future, just not for this release